pkg/pillar/cmd/zedrouter/metadata.go

```diff
@@ -148,6 +158,11 @@ func (z *zedrouter) makeMetadataHandler() http.Handler {
 		zedcloudCtx: &zedcloudCtx,
 	}
 	mux.Handle("/eve/v1/tpm/signer", signerHandler)
+	diagHandler := &diagHandler{
+		zedrouter:   z,
+		zedcloudCtx: &zedcloudCtx,
```

Comment:
zedcloudCtx is not needed inside diagHandler

Reply:
Fixed

Reply:
Fixed.